packages/wrangler/src/worker.ts

```diff
@@ -203,6 +203,10 @@ export interface CfDurableObjectMigrations {
 	}[];
 }
 
+export interface CfPlacement {
+	mode?: "smart" | "";
```

Comment:
It's optional and `"smart" | ""` because the worker upload API treats an empty string or undefined as off. I can simply to:

```ts
export interface CfPlacement {
    mode: "smart";
}
```

and just not add a placement to the worker metadata if the mode is off.